app/controllers/api/v1/transcriptions_controller.rb

```diff
@@ -64,6 +78,7 @@ def transcription_params
 
   def format_transcription(transcription)
     transcription.attributes.merge(
+      transcription_text: transcription.transcription_text.nil? ? "" : transcription.transcription_text,
```

Comment:
```suggestion
      transcription_text: transcription.transcription_text || "",
```
Not sure if we should be setting empty string for transcriptions that doesn't have a text 🤔

Reply:
You raise a valid point. The Faraday guard will throw an exception if the response is null, so there is no need to manipulate the translation object afterward. I will remove this change added. Thank you for highlighting this concern.